scripts/dts/python-devicetree/src/devicetree/edtlib.py

```diff
@@ -1321,6 +1321,97 @@ def gpio_hogs(self) -> list[ControllerAndData]:
 
         return res
 
+    @property
+    def maps(self) -> list[ControllerAndData]:
+        res: list[ControllerAndData] = []
+
+        def count_cells_num(node: dtlib_Node, specifier: str) -> int:
+            """
+            Calculate the number of cells in the node.
+            When calculating the number of interrupt cells,
+            add up the values of the address cells.
+            """
+
+            if node is None:
+                _err("node is None.")
```

Comment:
This should be impossible if the type checking is working properly. The only way `node` could be `None` is if its type annotation was `dtlib_Node | None`. The fact that the type is `dtlib_Node` already means it cannot be`None`. If that is not true, something is wrong with the type checking or the type annotation.

Reply:
Admittedly, this is redundant if there is type checking.
Removed.